src/Common/src/Interop/User32/Interop.QS.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum QS : uint
```

Comment:
Shouldn't this have a `[Flags]` attribute?

```suggestion
        [Flags]
        public enum QS : uint
```